viewer/db.js

```diff
@@ -154,10 +155,17 @@ Db.initialize = async (info, cb) => {
     console.log(`prefix:${internals.prefix} usersPrefix:${internals.usersPrefix}`);
   }
 
+  // build regular expressions for the user-specified extra query index patterns
```

Comment:
See the above comment for `queryExtraIndicesRegex`. Build regular expressions for the user-specified extra query index patterns.